constans/constants_utils.py

```diff
@@ -1,6 +1,6 @@
-NORTH = "w"
-SOUTH = "s"
-EAST = "d"
-WEST = "a"
-MOVE = "m"
-SHOOT = "z"
+NORTH = "NORTH"
+SOUTH = "SOUTH"
+EAST = "EAST"
+WEST = "WEST"
+MOVE = "MOVE"
+SHOOT = "SHOOT"
```

Comment:
Do we need these duplicate constants? Can we use the existing one in constants.py?

Reply:
No, we don't need it. It's a mistake.  I needed to change these constants because some tests were failing. 
I did not want to extend the PR.
I will delete these constants  and fix the tests